python/cudf/cudf/core/column/timedelta.py

```diff
@@ -227,6 +227,20 @@ def _binary_op_truediv(
 
         return lhs, rhs, out_dtype
 
+    def _make_copy_with_na_as_null(self):
+        """Return a copy with NaN values replaced with nulls."""
```

Comment:
Are `NaN` and `NaT` the same?
```suggestion
        """Return a copy with NaT values replaced with nulls."""
```

Reply:
From google: For datetime64[ns] types, `NaT `represents missing values.